lib/sing_for_needs/causes.ex

```diff
@@ -43,6 +43,28 @@ defmodule SingForNeeds.Causes do
     |> Repo.all()
   end
 
+  defp causes_query(%{scope: scope, limit: limit}) do
```

Comment:
You're not allowing the injection of any other query params here -- that's more of an observation than anything else, you don't seem to need them now.